ui/src/components/MyPollsPreview/MyPollsPreview.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render } from 'react-dom';
+import './MyPollsPreview.css';
+
+class MyPollsPreview extends React.Component {
+    render() {
+        const isFinished = (this.props.pollId != -1);
```

Comment:
You should add a TODO here so we can find it easier when we'll write the middleware:
```suggestion
        // TODO add check for "active" in particular poll's properties
        const isFinished = (this.props.pollId != -1);
```